yaml/xyz/openbmc_project/Inventory/Item/PCIeSlot.interface.yaml

```diff
@@ -98,6 +104,46 @@ enumerations:
           description: >
               Type of the PCIe slot is unknown
 
+    - name: Status
+      description: >
+          Possible link status
+      values:
+          - name: 'Operational'
+            description: >
+                Link is operational
+
+          - name: 'Degraded'
+            description: >
+                Link is degraded
+
+          - name: 'Unused'
```

Comment:
Unused is here twice

Reply:
Can we remove unused all together since it is not used ?
```
[Travis Pizel](https://app.slack.com/team/W8F5RSU8Y)  [4 hours ago](https://ibm-systems-power.slack.com/archives/C0Q6TQP5Z/p1650623780086079?thread_ts=1650399924.109139&cid=C0Q6TQP5Z)
For all of them, they're just simply not used.
```